aemedge/styles/styles.css

```diff
@@ -200,6 +200,29 @@ main pre {
   white-space: pre;
 }
 
+/* https://sap.frontify.com/document/223142#/-/links */
```

Comment:
Please remove this references, the are all public and just adding extra bytes

Reply:
Will do this in the other branch: https://github.com/urfuwo/hlx-test/tree/513-when-to-use